notebooks/book2/03/hierarchical_binom_rats.ipynb

```diff
@@ -0,0 +1,1831 @@
+{
```

Comment:
<p>English needs rewriting. Just say this</p><p>`</p><pre class="ql-syntax" spellcheck="false">We fit a hierarchical beta-binomial model to some count data derived from rat survival.  Based on&nbsp;https://docs.pymc.io/notebooks/GLM-hierarchical-binominal-model.html
</pre><p>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Correct URL: https://www.pymc.io/projects/examples/en/latest/generalized_linear_models/GLM-hierarchical-binomial-model.html